content/customer_promise.md

```diff
@@ -0,0 +1,52 @@
+#Customer promise
+
+Our customer promise tells you what we promise to do when you book a Pension Wise appointment.
+
+We will:
+
+##Booking your appointment
+
+- Aim to answer your call to our booking centre in 20 seconds or less
+- Explain clearly how to contact us, including booking a free 45-minute appointment
+- Offer you a choice of a telephone or face-to-face appointment within 14 days
+- Find a face-to-face appointment in a place convenient for you
+
+##After you book
+
+- Send you confirmation of your appointment
+- Tell you what will happen next, and by when
+- Tell you what to take to the appointment to get the best out of it
+- Send you reminders of your appointment
+
+##During your appointment
+
+- Give you the correct information
+- Explain things clearly and avoid jargon
+- Won’t recommend any products or tell you what to do with your money
+- Tell you where to go for help we can’t give you
+- Be helpful, impartial, polite, treat you fairly and with respect
+
+##After your appointment
+
+- Send you a written summary of the guidance you were given in your appointment
+- Use your feedback to improve how we do things
+
+##Personal information
+
+- Protect your personal information
+
+##Complaints
+
+- Tell you what to do next if you’re not satisfied with how you’ve been treated
+- Continuously improve our services to make things better for you
+
+##Accessibility
+
+- Make sure you can get a face-to-face appointment if you have a disability -- eg wheelchair access or in British Sign Language if you’re deaf
+- Send your summary in braille or large print if you’re blind or partially sighted
+
+^In return we need you to:^
+
+- be on time for your appointment or let us know if it needs to change
```

Comment:
Looking through the rest of the site, capitalising the first word would keep this consistent. I took the liberty of changing it here too.